media-source-respec.html

```diff
@@ -2332,6 +2332,69 @@ <h2>Examples</h2>
 
 &lt;video id="v" autoplay&gt; &lt;/video&gt;
 
+&lt;script&gt;
```

Comment:
This appears to begin the duplicative hiccup noted in the conversation (https://github.com/w3c/media-source/pull/265#issuecomment-828443753)  and should be fixed by the next commit in this sequence.

Reply:
Yes, it looks like this hiccup is fixed by the second commit.